core/src/main/java/org/elasticsearch/search/internal/ShardSearchLocalRequest.java

```diff
@@ -210,6 +219,9 @@ protected void innerReadFrom(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(Version.V_5_6_0)) {
             clusterAlias = in.readOptionalString();
         }
+        if (in.getVersion().onOrAfter(Version.V_6_0_0_beta1)) {
+            strict = in.readOptionalBoolean();
+        }
```

Comment:
serialization in SearchPhaseExecutionException.java uses 7.0 while this class uses 6.0, which version will this feature get in?